dotcom-rendering/src/components/LiveBlogBlocksAndAdverts.tsx

```diff
@@ -35,6 +72,10 @@ export const LiveBlogBlocksAndAdverts = ({
 	isLiveUpdate,
 	isInLiveblogAdSlotTest = false,
 }: Props) => {
+	const { renderingTarget } = useConfig();
+	const isWeb = renderingTarget === 'Web';
+	const isApps = renderingTarget === 'Apps';
+
 	if (!isInLiveblogAdSlotTest) {
```

Comment:
Commercial have now removed this test: https://github.com/guardian/frontend/pull/26670

As a result I think we'll lose ads on liveblogs and deadblogs in DCR for apps articles, so I'm going to merge #8962 to de-couple us from the test. This will also switch us back to the apps ad algorithm.

FYI @arelra